hack/lib/util.sh

```diff
@@ -437,6 +437,23 @@ kube::util::git_upstream_remote_name() {
     head -n 1 | awk '{print $1}'
 }
 
+# Ensures the current directory is a git tree for doing things like restoring or
+# validating godeps
+kube::util::ensure-git-tree() {
```

Comment:
it does, and was an artifact of my first attempt to make it so.. however it's actually necessary for this function to force creating the git tree as it if fails, then other steps will fail too. changed the function name.